src/Settings.php

```diff
@@ -109,6 +109,14 @@ class Settings
      */
     public $ignoreFails = false;
 
+    public function __construct()
+    {
+        // Detect console colors support
+        $this->colors = DIRECTORY_SEPARATOR === '\\'
```

Comment:
Can you rewrite this condition more readably? Because it looks too magic. For example you can extract parts of condition to variables with good names.